### PR TITLE
Replace 0-arg CP w/ eager instantiation

### LINKED ISSUE #143: Remove unnecessary zero-dependency computed property
this
https://github.com/bgentry/ember-apollo-client/blob/5296032ff0b10ac3bd9bebf6cca8f56946c90929/addon/mixins/base-query-manager.js#L8-L10

does not need to be a computed property. If you don't care about lazy instantiation, you could just
```js
{
   init() {
       this._super(...arguments);
       // Each instance gets its own query manager
       this.apollo = this.get('apolloService').createQueryManager();
   }
}
```
If you do care about laziness
```js
{
    _apolloQm: null, // Each instance gets its own query manager
   get apollo() {
       if (!this._apolloQm) { // Instantiate (and cache) on first ask
          this._apolloQm = this.get('apolloService').createQueryManager();
      }
      return this._apolloQm;
   }
}
```
In either case, existing consumers will not be affected. `Ember.Object#get` will work with all above approaches

### PR DESCRIPTION
**Pull request by mike-north**
Fixes #143

### CODE REVIEW

File: addon/mixins/base-query-manager.js [hunk 1]
```diff
@@ -4,8 +4,8 @@ import { computed } from '@ember/object';
 
 export default Mixin.create({
   apolloService: service('apollo'),
-
-  apollo: computed(function() {
-    return this.get('apolloService').createQueryManager();
-  }),
+  init() {
+    this._super(...arguments);
+    this.apollo = this.get('apolloService').createQueryManager();
```
Comment (bgentry) [inline addon/mixins/base-query-manager.js]: should this use a setter, i.e. `this.set("apollo", ...)` ?

Comment (mike-north) [reply addon/mixins/base-query-manager.js]: Yep, for consistency I should, although in an init hook you can do this safely.

File: addon/mixins/base-query-manager.js [hunk 2]
```diff
@@ -4,8 +4,8 @@ import { computed } from '@ember/object';
```
Comment (bgentry) [inline addon/mixins/base-query-manager.js]: need to drop the `computed` import to satisfy the linter

**Comment by bgentry** [comment]
Thanks @mike-north :v:

Feel free to PR yourself into the readme contributors list.

**Pull request merged**